draft-irtf-cfrg-vdaf.md

```diff
@@ -1557,12 +1566,15 @@ def prep_init(Prio3, verify_key, agg_id, _agg_param,
         dst,
         Prio3.Flp.QUERY_RAND_LEN
     )
-    joint_rand, k_joint_rand, k_joint_rand_share = [], None, None
+    joint_rand, k_joint_rand, k_joint_rand_part = [], None, None
     if Prio3.Flp.JOINT_RAND_LEN > 0:
         encoded = Prio3.Flp.Field.encode_vec(input_share)
-        k_joint_rand_share = Prio3.Prg.derive_seed(
+        k_joint_rand_part = Prio3.Prg.derive_seed(
             k_blind, dst + byte(agg_id) + encoded)
-        k_joint_rand = xor(k_hint, k_joint_rand_share)
+        k_joint_rand_parts = k_hint[:agg_id] + \
+                             [k_joint_rand_part] + \
```

Comment:
What do the enclosing `[]` denote on this line? `k_joint_rand_part` is already a `Bytes` (`def derive_seed(Prg, seed: Bytes, info: Bytes) -> bytes`), so can't it already be concatenated with the elements of `k_hint`?

Reply:
The plus operators here are acting as list concatenation -- the first and last arguments are slices of the k_hint list, so they are both lists of Bytes. The middle argument is a list literal, consisting of just the `k_join_rand_part` Bytes object.